src/main/java/core/basesyntax/service/impl/DataWriterImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.DataWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class DataWriterImpl implements DataWriter {
+
+    @Override
+    public boolean writeDataToFile(List<String> list, String filename) {
```

Comment:
```suggestion
    public boolean writeDataToFile(List<String> list, String fileName) {
```
use camelCase for naming